makefiles/Makefile.fsharp.mk

```diff
@@ -34,11 +40,17 @@ fsharp-help:
 .PHONY: fsharp # Build F# OR-Tools. Set environment variable FSHARP_DEBUG=1 for debug symbols.
 fsharp:
 ifneq ($(FSHARP_COMPILER_CHECK),)
-	$(FSHARP_COMPILER) $(FLAG_PREFIX)target:library $(FLAG_PREFIX)out:bin$S$(FSHARP_ORTOOLS_DLL_NAME).dll $(FLAG_PREFIX)platform:anycpu $(FLAG_PREFIX)nocopyfsharpcore $(FLAG_PREFIX)lib:bin $(FLAG_PREFIX)reference:$(BASE_ORTOOLS_DLL_NAME).dll $(FSHARP_DEBUG) $(FS_SIGNING_FLAGS) ortools$Sfsharp$S$(FSHARP_ORTOOLS_DLL_NAME).fsx
+	$(SED) -i -e "s/VVVV/$(OR_TOOLS_VERSION)/" ortools$Sfsharp$S$(FSHARP_ORTOOLS_DLL_ASSEMBLYINFO_FILE) ortools$Sfsharp$S$(FSHARP_ORTOOLS_NUSPEC_FILE)
+	$(FSHARP_COMPILER) $(FLAG_PREFIX)target:library $(FLAG_PREFIX)out:bin$S$(FSHARP_ORTOOLS_DLL_NAME).dll $(FLAG_PREFIX)platform:anycpu $(FLAG_PREFIX)nocopyfsharpcore $(FLAG_PREFIX)lib:bin $(FLAG_PREFIX)reference:$(BASE_ORTOOLS_DLL_NAME).dll $(FSHARP_DEBUG) $(FS_SIGNING_FLAGS) ortools$Sfsharp$S$(FSHARP_ORTOOLS_DLL_NAME).fsx ortools$Sfsharp$S$(FSHARP_ORTOOLS_DLL_ASSEMBLYINFO_FILE)
 else
 	$(error Cannot find '$(FSHARP_COMPILER)' command which is needed for build. Please make sure it is installed and in system path.)
 endif
 
+.PHONY: fsharp-build-nuget # Build Nuget Package for distribution.
```

Comment:
not sure about the target name. currently the "convention" is `xxx_archive`:
cc: Work in Progress (WIP) deb_archive
java: java_archive (tar.gz?) and (WIP maven_archive)
python: python_archive (tar.gz)  and pypi_archive pypi_upload (for pypi package manager)
csharp: nuget_archive nuget_upload
fsharp proposal: nuget_fsharp_archive fsharp_archive (tar.gz?)

Still open to suggestion (and breaking change) for a better convention if any...

note: nuget csharp management is still a work in progress (and i still don't master this part and relation with 
netstandard)
note: also notice the multiple sub traget to build the nuget package in csharp still not sure if it's the correct way to split the step BUT i would like to have the same workflow consistency for C#, netstandard and F# (at the end)
ps: no sure about the csharpfz_nuget_* target